### DISCUSSION TITLE
Test instrumentation process crashed. Check [PATH].txt for details

### DISCUSSION THREAD

**Discussion by Noxiver** [leancodepl/patrol#2062] [Q&A]
I installed patrol and then configured it according to the instructions. I run the test and anywhere in the crash, it doesn’t give me an error, but just sends me to look at a file (which doesn’t exist)

I use patrol Patrol CLI version: 2.2.2

Can someone help me with this problem?

**Comment by piotruela** [comment]
I replied on the Discord channel so closing the discussion here
